tools/scripts/javalabSpritesheetGenerators/README.md

```diff
@@ -0,0 +1,15 @@
+## For Creating a Single Spritesheet
+
+### Requirements
+
+- Python - An ability to run scripts locally
+- Pillow - IDE Extension
+- TabSave - Or any other site for downloading images from urls in bulk
+
+### Steps
+
+- Within neighborhood sprites google sheet, filter by category (i.e. vehicles, other, etc)
```

Comment:
Maybe a link to the sheet?  Or is it already well-known internally?

Reply:
Good call- thanks!